components/config/src/config/mod.rs

```diff
@@ -561,21 +561,24 @@ ignored_content = []
         let config_str = r#"
 title = "My site"
 base_url = "example.com"
-ignored_content = ["*.{graphml,iso}", "*.py?"]
+ignored_content = ["*.{graphml,iso}", "*.py?", "**/{target,temp_folder}"]
         "#;
 
         let config = Config::parse(config_str).unwrap();
         let v = config.ignored_content;
-        assert_eq!(v, vec!["*.{graphml,iso}", "*.py?"]);
+        assert_eq!(v, vec!["*.{graphml,iso}", "*.py?", "**/{target,temp_folder}"]);
 
         let g = config.ignored_content_globset.unwrap();
-        assert_eq!(g.len(), 2);
+        assert_eq!(g.len(), 3);
         assert!(g.is_match("foo.graphml"));
         assert!(g.is_match("foo.iso"));
         assert!(!g.is_match("foo.png"));
         assert!(g.is_match("foo.py2"));
         assert!(g.is_match("foo.py3"));
         assert!(!g.is_match("foo.py"));
+        assert!(g.is_match("foo/bar/target"));
+        assert!(g.is_match("foo/bar/baz/temp_folder"));
+        assert!(g.is_match("temp_folder"));
```

Comment:
can you add some more test examples?

```
assert!(g.is_match("my/isos/foo.iso"));
assert!(g.is_match("content/poetry/zen.py"));
```

Just to assert that single files glob still work without striping the prefix.

Reply:
`assert!(g.is_match("content/poetry/zen.py"));` actually won't match since the pattern is `"*.py?"` (another test case `assert!(!g.is_match("foo.py"));` confirms that.

But yes, I'll add the rest.